src/devices/cpu/i960/i960.cpp

```diff
@@ -2396,6 +2402,8 @@ void i960_cpu_device::device_reset()
 	m_r[I960_FP] = m_program.read_dword(m_PRCB+24);
 	m_r[I960_SP] = m_r[I960_FP] + 64;
 	m_rcache_pos = 0;
+
+	std::fill(std::begin(m_irq_line_state), std::end(m_irq_line_state), 0);
```

Comment:
Is this the physical pin state? Those don't get cleared by the chip at reset. And if they do, it's the surrounding hw that clears the pins.

Reply:
I believe they do indeed represent the physical pin states. I had more or less copied the approach used by sh2.cpp but your point about the surrounding hardware clearing the pins makes sense.